mixins/elasticsearch.yml

```
@@ -66,7 +66,7 @@ actions:
           exit 1
         fi
 
-        if ! [ $ec_deployment_mutualized ] && [ "$ec_deployment_es_endpoint" != null ]; then
+        if ! [ "$ec_deployment_mutualized" = "true" ] && [ "$ec_deployment_es_endpoint" != null ]; then
```

While we're at it, I think it could be nice to add a simple comment here to explain what we're doing (wait for CCI worflow's end in some cases but not others)

---

This may be more clear to read (no need to negate the conditions) :
```suggestion
        if [ "$ec_deployment_mutualized" = "false" ] || [ "$ec_deployment_es_endpoint" = "null" ]; then
```
I precise that this is not tested :)

---

Done